src/modules/75-cd/components/PipelineSteps/DeployEnvStep/DeployEnvStep.tsx

```diff
@@ -423,7 +423,7 @@ const DeployEnvironmentWidget: React.FC<DeployEnvironmentProps> = ({
           environmentRef: getEnvironmentRefSchema(getString)
         })}
       >
-        {formik => {
+        {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
        {(formik: unknown) => {
```